src/Components/Nav/Nav.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+const selectedNav = (isActive: boolean) => classNames(
```

Comment:
not blocker: 
some trick
you can accept here 
`const selectedNav = ({ isActive }: { isActive: boolean }) => classNames(`

and use like

```
  <NavLink
    to="/"
    className={selectedNav}
  >
    Home
  </NavLink>
```